pkg/archive/archive_unix_test.go

```diff
@@ -23,19 +23,6 @@ import (
 	"gotest.tools/v3/skip"
 )
 
-func TestCanonicalTarNameForPath(t *testing.T) {
-	cases := []struct{ in, expected string }{
-		{"foo", "foo"},
```

Comment:
I kept the removal of the tests as a separate commit; if there's concerns about not having these tests (they _should_ be covered by TestCanonicalTarName below), I can drop the second commit.